pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -403,23 +403,23 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsDialog* dialog, QWidget*
 			tr("Selects the quality at which screenshots will be compressed. Higher values preserve more detail for JPEG, and reduce file "
 			   "size for PNG."));
 
-		dialog->registerWidgetHelp(m_ui.stretchY, tr("Stretch Height"), tr("100%"),
-			tr("Stretches (> 100%) or squashes (< 100%) the vertical component of the display"));
+		dialog->registerWidgetHelp(m_ui.stretchY, tr("Vertical Stretch"), tr("100%"),
+			tr("Stretches (\074 100%) or squashes (\076 100%) the vertical component of the display."));
```

Comment:
I suppose this is to number it for context?

Reply:
The crude "<" within the second parentheses is causing (at least on the latest Windows version) the tooltip to stop reading from that point. I read some QT manuals and they said that by replacing the values with the octon UTF-8 versions, they should be read properly. Any suggestions are welcome, since I'm no programmer.

Reply:
Can you not to \< or \> to escape the string? That usually gets around that kind of thing.

Reply:
I can do that if you tell me it won't break the program.

Reply:
Can you not test it locally?

Reply:
Nope, I guess I should try the packages compiled by the checks.

Reply:
I do recommend being able to build locally so you can check your changes, especially if stuff might be a tighter squeeze in the UI.  We have a guide on how to get going https://github.com/PCSX2/pcsx2/wiki/Setting-up-the-PCSX2-repository-on-Windows

Reply:
It appears that `\<` / `\>` was not the answer.
![imagen](https://user-images.githubusercontent.com/5641681/220338743-b035d055-15d5-448d-9e6a-42fd42d0df4a.png)

Reply:
bummer, what about \\< ? :D

Reply:
if it doesn't work, that's fine, just a shame, it's so much more user friendly for translators to have actual symbols, surely there must be a way to do it.

Reply:
As for translators... QT seems to be swapping certain characters between code and the TS file, so maybe it's changed again while running lupdate.

Reply:
Found the way. Those characters needed to be converted into entities to make them display correctly. Unsurprisingly, this is also how they show up in the TS format.

Reply:
None of these seem to work for me:

With `\074` and `\076`:

![image](https://user-images.githubusercontent.com/33353403/220341318-61b47067-a7ee-4e83-bbd5-fae8e4058340.png)

With `\>` and `\<`:

![image](https://user-images.githubusercontent.com/33353403/220341717-b4c08b89-bb60-4c19-89f4-a6a145cef787.png)

Seems to work with `&gt;` and `&lt;` tho:

![image](https://user-images.githubusercontent.com/33353403/220347762-afc66280-a3d5-4b3b-8739-1ae8f39b3121.png)

Reply:
ah yes, since Qt is HTML based, you should be able to use URI codes :)